daemon/instructions.rs

```diff
@@ -36,13 +36,18 @@ pub fn handle_message(message: Message, sender: &Sender<Message>, state: &Shared
 /// Queues a new task to the state.
 /// If the start_immediately flag is set, send a StartMessage to the task handler
 fn add_task(message: AddMessage, sender: &Sender<Message>, state: &SharedState) -> Message {
-    let starting_status = if message.create_stashed {
+    let starting_status = if message.create_stashed || message.enqueue_at.is_some() {
         TaskStatus::Stashed
     } else {
         TaskStatus::Queued
     };
 
-    let task = Task::new(message.command, message.path, starting_status);
+    let task = Task::new(
+        message.command,
+        message.path,
+        starting_status,
+        message.enqueue_at
+    );
```

Comment:
I cannot comment on the correct line, but it would be awesome to get a custom success message for delayed tasks when adding as well :)

Reply:
Done